pages/vi/profiles/aksh4y.md

```diff
@@ -0,0 +1,28 @@
+# Akshay Sadarangani
+## Boston, MA / EST Windows 10
+
+![img](https://images.unsplash.com/photo-1496211398645-a5e6126505b2?ixlib=rb-0.3.5&s=f6089d385bf156c414e610bf72821858&auto=format&fit=crop&w=1924&q=80)
```

Comment:
Can you add some description instead of `img` ?

Reply:
Updated!